lm_eval/tasks/__init__.py

```diff
@@ -22,14 +24,10 @@ class TaskManager:
 
     def __init__(
         self,
-        verbosity="INFO",
```

Comment:
Per #2102 :
> Suggestion: Remove the feature completely or at least make setLevel optional with verbosity: str | None = None and if verbosity is not None: eval_logger.setLevel(getattr(logging, verbosity))

let's retain the `verbosity` arg here and in `evaluator.py` , have it default to `None`, and only call setlevel if verbosity is not None?